src/Sentry.Extensions.Logging/SentryLogger.cs

```diff
@@ -118,7 +117,7 @@ public bool IsEnabled(LogLevel logLevel)
             }
         }
 
-        private bool SendEvent(
+        private bool ShouldSendEvent(
```

Comment:
`l`: Maybe `ShouldCaptureEvent` is a better name, because the hub captures an event `_hub.CaptureEvent(@event)`

Reply:
makes sense